jobs/migrations/advanced_permissions_phase_2.go

```diff
@@ -26,7 +27,10 @@ func (p *AdvancedPermissionsPhase2Progress) ToJSON() string {
 
 func AdvancedPermissionsPhase2ProgressFromJSON(data io.Reader) *AdvancedPermissionsPhase2Progress {
 	var o *AdvancedPermissionsPhase2Progress
-	json.NewDecoder(data).Decode(&o)
+	err := json.NewDecoder(data).Decode(&o)
+	if err != nil {
+		mlog.Warn("Error decoding advanced permissions phase 2 progress", mlog.Err(err))
+	}
```

Comment:
Same comment as earlier. Let's not change this and `Decode` it at the call site.